isort/profiles.py

```diff
@@ -24,7 +24,8 @@
     "force_single_line": True,
     "force_sort_within_sections": True,
     "lexicographical": True,
-    "single_line_exclusions": ("typing",),
+    "line_length": 1000,
+    "single_line_exclusions": ("collections.abc", "typing"),
```

Comment:
Shouldn't this also list `typing_extensions`? https://google.github.io/styleguide/pyguide.html#2241-exemptions

Reply:
And `six.moves`. Done.